plugins/logging/pkg/gateway/stream.go

```diff
@@ -16,8 +17,13 @@ func (p *Plugin) StreamServers() []streamext.Server {
 		},
 		{
 			Desc:              &collogspb.LogsService_ServiceDesc,
-			Impl:              p.otelForwarder,
+			Impl:              p.otelForwarder.LogsForwarder,
 			RequireCapability: wellknown.CapabilityLogs,
 		},
+		{
+			Desc:              &coltracepb.TraceService_ServiceDesc,
+			Impl:              p.otelForwarder.TraceForwarder,
+			RequireCapability: wellknown.CapabilityTraces,
```

Comment:
this should be `wellknown.CapabilityLogs`, unless we really want to introduce logic inside the logging agent capability api to toggle logs/traces separately.

@dbason I think this is what you wanted for the initial pass to be for traces?

Reply:
Technically this field doesn't do anything yet, but it's reserved for future use, but yeah it should be `wellknown.CapabilityLogs`. If we want traces to be separate then we would make a separate plugin for it and then it would be its own capability. Inside the logging plugin it's all part of the logging capability.